webui/react/src/omnibar/sampleTree.ts

```diff
@@ -0,0 +1,196 @@
+import { displayHelp, parseIds, visitAction } from 'omnibar/actions';
```

Comment:
**nit:** rename file to something more like `decisionTree.ts`?

Reply:
I restructured them into `trees`  dir how does that sound?